code/Compass.php

```diff
@@ -290,15 +301,15 @@ public function rebuild($verbose = false)
             $this->rebuildDirectory($dir);
         } else {
             if ($verbose) {
-                echo "\nRebuilding all\n";
+                echo "\nRebuilding all<br>\n";
```

Comment:
Ok, I also thought this is a bad idea. But if you rebuild in browser by dev/compass/rebuild you get all the messages in one line
 I'll take it out and that "module skipping" below und make a new Pull.